src/app/chat/signal_handling.nim

```diff
@@ -1,7 +1,7 @@
 proc handleSignals(self: ChatController) =
   self.status.events.on(SignalType.Message.event) do(e:Args):
     var data = MessageSignal(e)
-    self.status.chat.update(data.chats, data.messages, data.emojiReactions)
+    self.status.chat.update(data.chats, data.messages, data.emojiReactions, data.communities)
```

Comment:
This proc keeps growing in the number of parameters it receives. Perhaps we should create an intermediate object so we pass a single parameter, or pass the message signal directly?